src/components/IconSpinner/IconSpinner.js

```diff
@@ -62,9 +64,14 @@ const DelayedSpinner = props => {
  * @returns {JSX.Element} SVG icon
  */
 const Spinner = props => {
-  const { delay, ...restOfProps } = props;
+  const { delay, intl, ...restOfProps } = props;
+  const ariaLabel = intl.formatMessage({ id: 'IconSpinner.screenreader.loading' });
 
-  return delay != null ? <DelayedSpinner {...props} /> : <IconSpinner {...restOfProps} />;
+  return delay != null ? (
+    <DelayedSpinner ariaLabel={ariaLabel} {...props} />
+  ) : (
+    <IconSpinner ariaLabel={ariaLabel} {...restOfProps} />
+  );
 };
 
-export default Spinner;
+export default injectIntl(Spinner);
```

Comment:
Use more hook-oriented **_useIntl_** instead.
We should remove all these injectIntl HOC's from the codebase.